src/main/java/eu/dissco/backend/controller/AnnotationController.java

```diff
@@ -76,7 +78,8 @@ public ResponseEntity<JsonApiListResponseWrapper> getLatestAnnotationsJsonRespon
 
   @GetMapping(value = "/{prefix}/{postfix}/{version}", produces = MediaType.APPLICATION_JSON_VALUE)
   public ResponseEntity<AnnotationResponse> getAnnotation(@PathVariable("prefix") String prefix,
-      @PathVariable("postfix") String postfix, @PathVariable("version") int version) {
+      @PathVariable("postfix") String postfix, @PathVariable("version") int version)
+      throws JsonProcessingException, NotFoundException {
```

Comment:
We should really change "postfix" to "suffix" at some point...

Reply:
Yes item is already on the backlog, see 
https://naturalis.atlassian.net/browse/DD-215